api/v1beta1/grafanadashboard_types.go

```diff
@@ -124,6 +125,16 @@ func (in *GrafanaDashboard) GrafanaContentStatus() *GrafanaContentStatus {
 	return &in.Status.GrafanaContentStatus
 }
 
+// GrafanaContentMetrics implements GrafanaContentResource
+func (in *GrafanaDashboard) GrafanaContentMetrics() GrafanaContentMetrics {
+	return GrafanaContentMetrics{
+		URLRequestCounter:                metrics.DashboardUrlRequests,
+		GrafanaComRevisionRequestCounter: metrics.GrafanaComApiRevisionRequests,
+	}
+}
+
+var _ GrafanaContentResource = &GrafanaDashboard{}
```

Comment:
Since you're returning a static value here, I don't think we need this interface

Reply:
Sorry, marked the wrong lines. I was referencing the `GrafanaContentMetrics` but this is gone in the newest revision :)